content/sensu-go/6.3/observability-pipeline/observe-schedule/rule-templates.md

```diff
@@ -0,0 +1,471 @@
+---
+title: "Rule templates reference"
+linkTitle: "Rule Templates Reference"
+reference_title: "Rule templates"
+type: "reference"
+description: "Sensu's rule templates are resources that Sensu applies to business service components. The rules define the conditions under which Sensu will consider a service component online, degraded, or offline. Read this reference to learn more about Sensu's rule templates for business service management."
+weight: 70
+version: "6.3"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.3:
+    parent: observe-schedule
+---
+
+**COMMERCIAL FEATURE**: Access business service management, including rule templates, in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][4].
+
+Rule templates are resources that Sensu applies to business service [components][3].
+The rules define the conditions under which Sensu will consider a service component online, degraded, or offline.
```

Comment:
I think that's correct at a high level, but in reality it's even more general: the rules emit an event you can completely customize. The rules get a selection of events (as defined by the Service Component's Query) as input and they can do what they want with this information.